brave/brave-core
# feat(wallet): DApp Connection Settings V2

## Description 
Built out the new `DApp Connection Settings` for the Wallet `Panel V2`


Resolves <https://github.com/brave/brave-browser/issues/32035>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to any `DApp` and `Connect` your Wallet
2. Open the Wallet `Panel` and then open the `DApp Connection Settings`
3. Test and make sure the `Connection` status are correct
4. Test `Disconnecting` and `Connecting`
5. Test switching `Accounts`
6. Test switching `Networks`

https://github.com/brave/brave-core/assets/40611140/6b21e0a1-152d-468e-8421-81c1f19f0bcb
